### PR TITLE
Changed button to text

### PR DESCRIPTION
**Pull request by drakeloud**
Screen Shot 2019-10-19 at 11 58 32 AM

Updated the trashcan icon to delete text.

Addresses feature #4027

### CODE REVIEW

File: src/resources/styles.ts [hunk 1]
```diff
@@ -71,6 +71,10 @@ export const haStyleDialog = css`
     --mdc-theme-primary: var(--google-red-500);
   }
 
+  .paper-dialog-buttons .warning.left-align {
+    margin-right: auto;
+  }
```
Comment (bramkragten) [inline src/resources/styles.ts]: Move this to `hui-edit-views` `static get styles()`, there is a `.delete` class there that was used for the button, should clean that up and replace with the new styles.

File: src/translations/en.json [hunk 2]
```diff
@@ -502,6 +502,7 @@
     "common": {
       "loading": "Loading",
       "cancel": "Cancel",
+      "delete_view": "Delete View",
```
Comment (bramkragten) [inline src/translations/en.json]: https://github.com/home-assistant/home-assistant-polymer/blob/59989a902ce1136cdcb3afa6fb303bb059f2eb05/src/translations/en.json#L1386

Comment (bramkragten) [reply src/translations/en.json]: This should not be in `common` but in the section of lovelace and the views, there actually already is a key for `delete` there.

Comment (drakeloud) [reply src/translations/en.json]: @bramkragten I didn't realize those existed in en.json.

**Comment by drakeloud** [comment]
@bramkragten I have updated the PR with the styles being moved, and changing what the translation values are.

**Review by bramkragten — APPROVED** [review]

**Pull request merged**